cmd/snapd-apparmor/main.go

```diff
@@ -44,16 +44,13 @@ import (
 
 	"github.com/snapcore/snapd/dirs"
 	"github.com/snapcore/snapd/logger"
+	"github.com/snapcore/snapd/release"
 	apparmor_sandbox "github.com/snapcore/snapd/sandbox/apparmor"
 	"github.com/snapcore/snapd/snapdtool"
 )
 
-func isWSL() bool {
-	if output, err := exec.Command("systemd-detect-virt", "--container").Output(); err == nil {
-		virt := strings.TrimSpace(string(output))
-		return virt == "wsl"
-	}
-	return false
+var isWsl = func() bool {
```

Comment:
By convention, initialisms are written in a consistent case

```suggestion
var isWSL = func() bool {
```